spec/decorators/application_decorator_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "rails_helper"
+
+RSpec.describe ApplicationDecorator, type: :decorator do
+  describe "#object" do
+    it "exposes the decorated object" do
+      obj = Class.new
```

Comment:
This is a bit nitpicky, but when I saw `Class.new`, I wondered if it's significant that it's a class instance and had to read the code to make sure it wasn't. `Object.new` would avoid this confusion.

Reply:
adding this to the list of things to change in a following PR :D

Reply:
Sure, no rush.